oie changes for registration hooks (#2354)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

}
```

## Enabling a Registration Inline Hook for Profile Enrollment

<ApiLifecycle access="ie" /><br>

> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.

To activate the Inline Hook, you first need to register your external service endpoint with Okta; see [Inline Hook Setup](/docs/concepts/inline-hooks/#inline-hooks_setup).

You then need to associate the registered Inline Hook with your Profile Enrollment policy. (For information on configuring a Profile Enrollment policy, see [Enable and configure a profile enrollment policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm).)

1. Go to **Security > Profile Enrollment**

> 1. Click the pencil icon to edit the policy for which you want your registration hook to be associated with.

Reviewer: ```suggestion
1. Click the pencil icon to edit the policy and associate it with your Registration Inline Hook.
```

> 

Reviewer: ```suggestion
1. In **Enrollment Settings**, click the More Options icon and then select **Edit**. If not already selected, click **Allowed** in the **For new users** section.
```

> 1. Select your hook from the dropdown menu for **Use the following inline hook** under the options for **For new users**. If you have created multiple Registration Inline Hooks, you should see all of them displayed here.

Reviewer: ```suggestion
1. Select your hook from the drop-down menu for **Use the following inline hook** under the options for **For new users**. If you have created multiple Registration Inline Hooks, you should see all of them displayed here.
```

}
```

## Enabling a Registration Inline Hook for Profile Enrollment

<ApiLifecycle access="ie" /><br>

> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.

To activate the Inline Hook, you first need to register your external service endpoint with Okta; see [Inline Hook Setup](/docs/concepts/inline-hooks/#inline-hooks_setup).

You then need to associate the registered Inline Hook with your Profile Enrollment policy. (For information on configuring a Profile Enrollment policy, see [Enable and configure a profile enrollment policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm).)

1. Go to **Security > Profile Enrollment**

> 1. Click the pencil icon to edit the policy and associate it with your Registration Inline Hook.

Reviewer 2: Are these numbered items supposed to increment?

Author: they become incremental when the markdown is rendered
